### PR TITLE
Make sure '0' edge case is properly handled by is_json()

### PR DESCRIPTION
**Pull request by kirtangajjar**

### CODE REVIEW

File: php/utils.php [hunk 1]
```diff
@@ -1406,7 +1406,7 @@ function esc_sql_ident( $idents ) {
  * @return bool Whether the provided string is a valid JSON representation.
  */
 function is_json( $argument, $ignore_scalars = true ) {
-	if ( empty( $argument ) || ! is_string( $argument ) ) {
+	if ( ! is_string( $argument ) || '' ===  $argument ) {
```
Comment (gitlost) [inline php/utils.php]: Removing the extra space after the equals will satisfy the wpcs sniff, so `'' ===  $argument` goes to `'' === $argument` (hard to see on github!).

**Comment by gitlost** [comment]
Was https://github.com/wp-cli/wp-cli/pull/4634

File: php/utils.php (hunk 1 above)
Comment (kirtangajjar) [reply php/utils.php]: Oh! Sorry I didn't notice it.

**Comment by gitlost** [comment]
Thanks very much @kirtangajjar !

**Pull request merged**